nBTC/sources/storage.move

```diff
@@ -0,0 +1,147 @@
+module nbtc::storage;
+
+use ika_dwallet_2pc_mpc::coordinator_inner::DWalletCap;
+use sui::table::{Self, Table};
+
+public struct DWalletMetadata has store {
+    // TODO: change to taproot once Ika will support it
+    public_key: vector<u8>, // ecdsa public key
+    lockscript: vector<u8>, // lock script for this dwallet
+    script_type: u8, // script type, not sure we need this
+    active: bool, // active flag
```

Comment:
I think this is not needed here - we remeber active dwallet ID in nbtc.

Reply:
yes 👍🏼

Reply:
removed!